src/components/calcite-tooltip/calcite-tooltip.tsx

```diff
@@ -187,57 +199,56 @@ export class CalciteTooltip {
     );
   }
 
-  getBoundariesElement(): HTMLElement {
-    const { boundariesElement } = this;
+  getModifiers(): Partial<Modifier<any>>[] {
+    const { arrowEl, offsetDistance, offsetSkidding } = this;
 
-    return (
-      (typeof boundariesElement === "string"
-        ? document.getElementById(boundariesElement)
-        : boundariesElement) || null
-    );
-  }
+    const arrowModifier: Partial<Modifier<any>> = {
+      name: "arrow",
+      enabled: true,
+      options: {
+        element: arrowEl
+      }
+    };
 
-  getModifiers(): Popper.Modifiers {
-    const { _boundariesElement } = this;
-
-    return {
-      preventOverflow: {
-        enabled: true,
-        boundariesElement: _boundariesElement || "viewport",
-        escapeWithReference: true
-      },
-      flip: {
-        enabled: true,
-        boundariesElement: _boundariesElement || "viewport",
-        flipVariationsByContent: true
+    const offsetModifier: Partial<Modifier<any>> = {
+      name: "offset",
+      enabled: true,
+      options: {
+        offset: [offsetSkidding, offsetDistance]
       }
     };
+
+    return [arrowModifier, offsetModifier];
+  }
+
+  updatePopper(popper: Popper): void {
```

Comment:
Would it be possible to create helpers for `getModifiers` and `updatePopper`? Looks like `getModifiers` may differ based on the component.

Reply:
I can create helpers for updatePopper and createPopper but modifiers are different.